src/ensembl/src/content/app/help/Help.scss

```diff
@@ -5,10 +5,17 @@
 }
 
 .appBar {
-  display: flex;
+  display: grid;
   align-items: center;
   height: 80px;
   padding: 0 44px; 
+
+  grid-template-columns: 100px auto;
+
+  .CommunicationPanelIcon{
```

Comment:
we usually start names of CSS classes with a lower-case letter

Reply:
That's done